lms/djangoapps/verify_student/management/commands/tests/test_verify_student.py

```diff
@@ -28,78 +25,70 @@
 # Lots of patching to stub in our own settings, and HTTP posting
 @patch.dict(settings.VERIFY_STUDENT, FAKE_SETTINGS)
 @patch('lms.djangoapps.verify_student.models.requests.post', new=mock_software_secure_post)
-class TestVerifyStudentCommand(MockS3BotoMixin, TestCase):
+class TestVerifyStudentCommand(MockS3BotoMixin, TestVerificationBase):
     """
     Tests for django admin commands in the verify_student module
     """
-    def create_and_submit(self, username):
-        """
-        Helper method that lets us create new SoftwareSecurePhotoVerifications
-        """
-        user = UserFactory.create()
-        attempt = SoftwareSecurePhotoVerification(user=user)
-        user.profile.name = username
-        attempt.upload_face_image("Fake Data")
-        attempt.upload_photo_id_image("More Fake Data")
-        attempt.mark_ready()
-        attempt.submit()
-        return attempt
 
     def test_retry_failed_photo_verifications(self):
         """
         Tests that the task used to find "must_retry" SoftwareSecurePhotoVerifications
         and re-submit them executes successfully
         """
         # set up some fake data to use...
-        self.create_and_submit("SuccessfulSally")
+        self.create_upload_and_submit_attempt_for_user()
         with patch('lms.djangoapps.verify_student.models.requests.post', new=mock_software_secure_post_error):
-            self.create_and_submit("RetryRoger")
+            self.create_upload_and_submit_attempt_for_user()
         with patch('lms.djangoapps.verify_student.models.requests.post', new=mock_software_secure_post_error):
-            self.create_and_submit("RetryRick")
+            self.create_upload_and_submit_attempt_for_user()
```

Comment:
Can't these two calls be combined in one mock? They are same, so the block execution will have its effect.